src/pal/prebuilt/inc/mscoree.h

```diff
@@ -281,7 +281,11 @@ DECLARE_DEPRECATED void STDMETHODCALLTYPE CorExitProcess(int exitCode);
 DEPRECATED_CLR_STDAPI LoadStringRC(UINT iResouceID, _Out_writes_z_(iMax) LPWSTR szBuffer, int iMax, int bQuiet);
 typedef HRESULT (__stdcall *FLockClrVersionCallback) ();
 DEPRECATED_CLR_STDAPI LockClrVersion(FLockClrVersionCallback hostCallback,FLockClrVersionCallback *pBeginHostSetup,FLockClrVersionCallback *pEndHostSetup);
+/*
+On Unix this depricated function produces declaration conflict with another function with the same name from DbgShim.
+We don't expect depricated functions to be used on non-Windows anyway.
 DEPRECATED_CLR_STDAPI CreateDebuggingInterfaceFromVersion(int iDebuggerVersion, LPCWSTR szDebuggeeVersion, IUnknown ** ppCordb);
+*/
```

Comment:
This file is auto generated. We do not want to make this manual edit every time the file is refreshed. Could you please find a way how to make this work without manual edits to this auto-generated file?